Jint.Tests.PublicInterface/InteropTests.SystemTextJson.cs

```diff
@@ -2,19 +2,74 @@
 using System.Text.Json.Nodes;
 using Jint.Native;
 using Jint.Runtime.Interop;
-
+using System.Text.Json;
 namespace Jint.Tests.PublicInterface;
 
+#if NET8_0_OR_GREATER
```

Comment:
I don't think this is supposed to be here, is it? At least I don't see any .NET 8 specific code at a glance?

Reply:
``` c#
            var valueKind = jsonValue.GetValueKind();
```
This line ,the `GetValueKind()` is warning not support net462

Reply:
Yes, that method was introduced with .NET 8...but the available NuGet package that ships `System.Text.Json` still supports .NET Framework 4.6.2 upwards; so it _should_ work there as well. If this wasn't the case, the old code should've errored out already too.

Granted, I don't have any STJ interop scenarios on .NET Framework at the moment, but I still use this in a .NET 6 integration that is waiting to get upgraded to .NET 8. And as far as I can tell, things _should_ work there if you override the package version. I'd have to test this though, and that might be a while.

Maybe gate the converter behind this, and leave the old code as alternative for anyone that isn't on .NET 8 yet? Remember those tests serve as documentation, and having them work for _fewer_ people could be a problem (at least it would be in my head, not sure if others disagree.)

Reply:
> Remember those tests serve as documentation, and having them work for fewer people could be a problem (at least it would be in my head, not sure if others disagree.)

Yes, I agree, I think it's enough to provide a concrete example that people can use to transform different complex objects, so we shouldn't worry too much about whether it works in all versions，people will always have a way to make it work😁